### PR TITLE
refactor(db): Remove account unlock related code.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
@jrgm thinks that the `lockedAt` column on the account table does not take
up enough space to warrant the effort needed to remove it.

@philbooth - mind having an initial look at this to see if I'm in the right track?

### CODE REVIEW

**Comment by philbooth** [comment]
> @philbooth - mind having an initial look at this to see if I'm in the right track?

You're on the right track (imho). With test changes I think this will be good to go!

**Comment by shane-tomlinson** [comment]
@rfk and @jrgm - r?

File: lib/db/schema/patch-032-031.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+-- -- the codes to unlock an account
+-- CREATE TABLE accountUnlockCodes (
+--   uid BINARY(16) PRIMARY KEY NOT NULL,
+--   unlockCode BINARY(16) NOT NULL
+-- ) ENGINE=InnoDB;
+--
+-- DROP PROCEDURE `resetAccunt_7`;
```
Comment (vladikoff) [inline lib/db/schema/patch-032-031.sql]: typo in procedure name

**Comment by rfk** [comment]
LGTM

**Comment by rfk** [comment]
-> @jrgm

**Comment by rfk** [comment]
@jrgm says r+ in meeting

**Pull request merged**